src/SimplCommerce.WebHost/Themes/CozaStore/Views/Shared/_SelectLanguagePartial.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@using Microsoft.AspNetCore.Localization
```

Comment:
@thiennn IMHO we could move this to localization module to use it everywhere, instead of duplicated the code each time

Reply:
noted. Thanks